### PR TITLE
New package: 0ad-0.0.23

### PR DESCRIPTION
**Pull request by Hoshpak**

### CODE REVIEW

**Comment by Johnnynator** [comment]
Works fine on my x86_64-musl machine.
If you want to have `xbps-src update-check` working just add this `pattern='href="0ad-\K[\d\.]*+(?=-alpha-unix-build.tar.xz")'`

**Comment by Hoshpak** [comment]
Thanks. I added the update pattern.

File: srcpkgs/0ad-data/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Template file for '0ad-data'
+pkgname=0ad-data
+version=0.0.23
+revision=1
+wrksrc="0ad-${version}-alpha"
+short_desc="Historically-based real-time strategy game (data files)"
+maintainer="Helmut Pozimski <helmut@pozimski.eu>"
+license="CC-BY-SA-3.0"
+homepage="https://play0ad.com"
+noarch=yes
```
Comment (maxice8) [inline srcpkgs/0ad-data/template]: unde rrevision=

Comment (Hoshpak) [reply srcpkgs/0ad-data/template]: I'm not sure what you mean, could you be more specific?

**Comment by maxice8** [comment]
On Sat, Jun 23, 2018 at 02:28:40PM -0700, Helmut Pozimski wrote:
> Hoshpak commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,20 @@
> +# Template file for '0ad-data'
> +pkgname=0ad-data
> +version=0.0.23
> +revision=1
> +wrksrc="0ad-${version}-alpha"
> +short_desc="Historically-based real-time strategy game (data files)"
> +maintainer="Helmut Pozimski <helmut@pozimski.eu>"
> +license="CC-BY-SA-3.0"
> +homepage="https://play0ad.com"
> +noarch=yes
> 
> I'm not sure what you mean, could you be more specific?
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/144#discussion_r197622936

noarch=yes should be below the revision= field.

**Comment by Hoshpak** [comment]
So directly below? I understood that it should be somewhere below the revision field, not necessarily the next one. Commit amended.

**Comment by maxice8** [comment]
On Sat, Jun 23, 2018 at 02:35:19PM -0700, Helmut Pozimski wrote:
> So directly below? I understood that it should be somewhere below the revision field, not necessarily the next one. Commit amended.
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/144#issuecomment-399712137

yes, directly below.

File: srcpkgs/0ad-data/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+pkgname=0ad-data
+version=0.0.23
+revision=1
+noarch=yes
+wrksrc="0ad-${version}-alpha"
+short_desc="Historically-based real-time strategy game (data files)"
+maintainer="Helmut Pozimski <helmut@pozimski.eu>"
+license="CC-BY-SA-3.0"
+homepage="https://play0ad.com"
+distfiles="http://releases.wildfiregames.com/0ad-${version}-alpha-unix-data.tar.xz"
```
Comment (travankor) [inline srcpkgs/0ad-data/template]: distfile actually supports ssl https://releases.wildfiregames.com/

File: srcpkgs/0ad/template [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+revision=1
+wrksrc="${pkgname}-${version}-alpha"
+hostmakedepends="pkg-config perl cmake python"
+makedepends="boost-devel libcurl-devel libenet-devel gloox-devel icu-devel nspr-devel libogg-devel libvorbis-devel libXcursor-devel libxml2-devel miniupnpc-devel libopenal-devel SDL2-devel zlib-devel libpng-devel wxWidgets-devel libsodium-devel"
+depends="${pkgname}-data"
+short_desc="Historically-based real-time strategy game"
+maintainer="Helmut Pozimski <helmut@pozimski.eu>"
+license="GPL-2.0-or-later, CC-BY-SA-3.0, MPL-2.0, MIT"
+homepage="https://play0ad.com"
+distfiles="http://releases.wildfiregames.com/${pkgname}-${version}-alpha-unix-build.tar.xz"
```
Comment (travankor) [inline srcpkgs/0ad/template]: https

File: srcpkgs/0ad/template [hunk 4]
```diff
@@ -0,0 +1,38 @@
+# Template file for '0ad'
+pkgname=0ad
+version=0.0.23
+revision=1
+wrksrc="${pkgname}-${version}-alpha"
+hostmakedepends="pkg-config perl cmake python"
+makedepends="boost-devel libcurl-devel libenet-devel gloox-devel icu-devel nspr-devel libogg-devel libvorbis-devel libXcursor-devel libxml2-devel miniupnpc-devel libopenal-devel SDL2-devel zlib-devel libpng-devel wxWidgets-devel libsodium-devel"
```
Comment (travankor) [inline srcpkgs/0ad/template]: new line at 80 characters

**Pull request merged**